frontend/projects/upgrade/src/app/core/experiments/experiments.data.service.ts

```diff
@@ -67,13 +67,22 @@ export class ExperimentDataService {
   }
 
   exportExperimentInfo(experimentId: string, email: string) {
+    let experimentInfoParams = new HttpParams();
```

Comment:
let -> const

Reply:
using experimentInfoParams again in next line